HousingFinanceInterimApi/V1/Controllers/AssetController.cs

```diff
@@ -25,6 +23,7 @@ public AssetController(IAssetGateway assetGateway)
         [ProducesResponseType(StatusCodes.Status500InternalServerError)]
         [HttpPatch]
         [Route("{propertyReference}")]
+        [AuthorizeEndpointByGroups("ASSET_ADMIN_GROUPS")]
```

Comment:
This is just an environment variable name, you also need to get it configured within the `serverless.yml` file. Otherwise there's going to be nothing to bundle & there won't be any groups for the authorizer to look up.